crates/ruff_wasm/src/lib.rs

```diff
@@ -49,34 +48,17 @@ export interface Diagnostic {
 };
 "#;
 
-#[derive(Serialize)]
-struct ExpandedMessage {
-    code: SerializeRuleAsCode,
```

Comment:
Is there a way to deserialize a noqa code back to a rule? Although I'm not sure if that's important.

Reply:
No, I don't believe so, not right now at least.